### ISSUE TITLE
on loading Deck, show board list (don't collapse it)

### ISSUE THREAD

**Issue by alexanderdd** [nextcloud/deck#2590]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

**Describe the bug**
on loading Deck, the upcoming cards view is shown with the board list sidebar collapsed. This confuses users, they don't know how to get to other boards, or that other boards even exist.

**To Reproduce**
1. Click on Deck in the top menu bar
2. see the upcoming cards view
3. be confused

**Expected behavior**
On loading Deck, show upcoming cards view, but with the board list sidebar **open**.

Alternative: on loading, show the board list sidebar briefly, then slide it away. So the user is aware that it is there.

**Comments**
- The board list sidebar gives essential orientation to the user, shows him/her where he/she is, and what else exists.
- The collapsing of the sidebar should be a deliberate user action (clicking on the hamburger icon).
- I assume that the most frequent use of Deck is clicking on Deck in top menu, then going to a board. For this action, now you need three clicks, if my proposal is implemented it goes back to two clicks (was two clicks in the past).
- I can not think of an application that has a sidebar but collapses it by default -- it is always open in the beginning.
- I think this classifies as a UX bug (not a feature), that's why I entered it as a bug.

**Client details:**
 - OS: Linux Mint
 - Browser Firefox 
 - Version 83
 - Device: desktop

**Server**
Nextcloud 19.0.4
Deck 1.2.1

**Comment by juliushaertl** [comment]
Yes, as discussed with @jancborchardt in our design call yesterday, we should always show the board list at top level and only have the archived boards collapsed.

**Comment by alexanderdd** [comment]
still there in Deck 1.2.3

@juliushaertl for me this is a bug, not an enhancement. Users tell me that they are confused and don't know what to do.

**Comment by alexanderdd** [comment]
@juliushaertl yes, remembering the last state works as you discribed. But for new users it is collapsed.

So maybe rename this issue "on first use of Deck, show board list (don't collapse it)"

**Comment by jnny** [comment]
Just chiming in to second @alexanderdd's request, as it is indeed quite confusing for new users!

**Comment by stefan-niedermann** [comment]
@jnny Please use **[GitHub reactions](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/)** 👍 to show that you are affected by the same issue. Please don't comment if you have no relevant information to add.

**Comment by alexanderdd** [comment]
What other info do you need? Or remove the "needs info" label @juliushaertl

**Comment by alvarofranz** [comment]
I wonder how this doesn't have more activity. It's really frustrating that the Upcoming Cards view shows even when there are no Upcoming cards at all. There should be an option to set a specific board as the default view.

**Comment by alexanderdd** [comment]
> What other info do you need? Or remove the "needs info" label @juliushaertl

@juliushaertl Still waiting what info is needed? Or please remove the "needs info" label. 

Or can someone else remove the label?

@alvarofranz your wish should be discussed in https://github.com/nextcloud/deck/issues/4372 This issue here is about something else (always show board list sidebar)

**Comment by alvarofranz** [comment]
Yeah, I commented here because that one was closed to reference this one. Though... yes, I agree, the topic is different.
